meta-xt-prod-devel-rcar-control/recipes-guests/domd/files/domd-h3ulcb-4x2g-kf.cfg

```diff
@@ -1,44 +1,43 @@
 # =====================================================================
 # DomD guest configuration
 # =====================================================================
-
 seclabel='system_u:system_r:domD_t'
-
 # Guest name
 name = "DomD"
-
 # Kernel image to boot
 kernel = "/usr/lib/xen/boot/linux-domd"
-
 device_tree = "/usr/lib/xen/boot/domd.dtb"
-
 # Kernel command line options
 # Uncomment this when using nfs as a boot device
 #extra = "root=/dev/nfs nfsroot=192.168.1.100:/srv/domd,vers=3 ip=dhcp rw rootwait console=hvc0 pvrsrvkm.DriverMode=0 xt_page_pool=67108864 xt_cma=100663296"
 # Uncomment this when using block device as a boot device
 extra = "root=/dev/STORAGE_PART2 rw rootwait console=hvc0 pvrsrvkm.DriverMode=0 xt_page_pool=67108864 xt_cma=100663296"
-
 # Initial memory allocation (MB)
 memory = 2048
-
 # Number of VCPUS
 vcpus = 4
-
 on_crash = 'preserve'
-
 # This is autogenerated config part
-
 dt_compatible = [ "shimafuji,kingfisher", "renesas,h3ulcb", "renesas,r8a7795" ]
-
 # Clocks and regulators are needed and are in the device tree root,
 # but we do not want to copy all root nodes: handle these one by one
 dt_passthrough_nodes = [
     "/audio_clk_a",
     "/audio_clk_b",
     "/audio_clk_c",
     "/can",
+    "/opp_table0",
```

Comment:
Please take domd-h3ulcb-4x2g-kf.cfg from old prod-devel master HEAD. There are no nodes like `opp_tableX` or `firmware`. And mentioned config (from old prod-devel master) is proven to work on new prod-devel-rcar KF.